### ISSUE TITLE
Found some debug log error

### ISSUE THREAD

**Issue by apeksha10** [TycheSoftwares/Woocommerce-abandon-cart-lite#591]
**Describe the bug**
Found some debug log error

[25-Jan-2020 05:45:07 UTC] PHP Deprecated:  The behavior of unparenthesized expressions containing both '.' and '+'/'-' will change in PHP 8: '+'/'-' will take a higher precedence in C:\wamp64\www\wordpress\wp-content\plugins\woocommerce-abandoned-cart\includes\classes\class-wcal-dashboard-report.php on line 359
[25-Jan-2020 05:45:07 UTC] PHP Stack trace:
[25-Jan-2020 05:45:07 UTC] PHP   1. {main}() C:\wamp64\www\wordpress\wp-admin\admin.php:0
[25-Jan-2020 05:45:07 UTC] PHP   2. do_action() C:\wamp64\www\wordpress\wp-admin\admin.php:254
[25-Jan-2020 05:45:07 UTC] PHP   3. WP_Hook->do_action() C:\wamp64\www\wordpress\wp-includes\plugin.php:478
[25-Jan-2020 05:45:07 UTC] PHP   4. WP_Hook->apply_filters() C:\wamp64\www\wordpress\wp-includes\class-wp-hook.php:312
[25-Jan-2020 05:45:07 UTC] PHP   5. woocommerce_abandon_cart_lite->wcal_menu_page() C:\wamp64\www\wordpress\wp-includes\class-wp-hook.php:288
[25-Jan-2020 06:00:06 UTC] PHP Notice:  Object of class stdClass could not be converted to number in C:\wamp64\www\wordpress\wp-content\plugins\woocommerce-abandoned-cart\cron\wcal_send_email.php on line 73

[25-Jan-2020 06:00:06 UTC] PHP Stack trace:
[25-Jan-2020 06:00:06 UTC] PHP   1. {main}() C:\wamp64\www\wordpress\wp-admin\admin-ajax.php:0
[25-Jan-2020 06:00:06 UTC] PHP   2. do_action() C:\wamp64\www\wordpress\wp-admin\admin-ajax.php:175
[25-Jan-2020 06:00:06 UTC] PHP   3. WP_Hook->do_action() C:\wamp64\www\wordpress\wp-includes\plugin.php:478
[25-Jan-2020 06:00:06 UTC] PHP   4. WP_Hook->apply_filters() C:\wamp64\www\wordpress\wp-includes\class-wp-hook.php:312
[25-Jan-2020 06:00:06 UTC] PHP   5. ActionScheduler_AsyncRequest_QueueRunner->maybe_handle() C:\wamp64\www\wordpress\wp-includes\class-wp-hook.php:288
[25-Jan-2020 06:00:06 UTC] PHP   6. ActionScheduler_AsyncRequest_QueueRunner->handle() C:\wamp64\www\wordpress\wp-content\plugins\woocommerce\includes\libraries\wp-async-request.php:147
[25-Jan-2020 06:00:06 UTC] PHP   7. do_action() C:\wamp64\www\wordpress\wp-content\plugins\woocommerce-abandoned-cart\includes\libraries\action-scheduler\classes\ActionScheduler_AsyncRequest_QueueRunner.php:52
[25-Jan-2020 06:00:06 UTC] PHP   8. WP_Hook->do_action() C:\wamp64\www\wordpress\wp-includes\plugin.php:478
[25-Jan-2020 06:00:06 UTC] PHP   9. WP_Hook->apply_filters() C:\wamp64\www\wordpress\wp-includes\class-wp-hook.php:312
[25-Jan-2020 06:00:06 UTC] PHP  10. ActionScheduler_QueueRunner->run() C:\wamp64\www\wordpress\wp-includes\class-wp-hook.php:288
[25-Jan-2020 06:00:06 UTC] PHP  11. ActionScheduler_QueueRunner->do_batch() C:\wamp64\www\wordpress\wp-content\plugins\woocommerce-abandoned-cart\includes\libraries\action-scheduler\classes\ActionScheduler_QueueRunner.php:120
[25-Jan-2020 06:00:06 UTC] PHP  12. ActionScheduler_QueueRunner->process_action() C:\wamp64\www\wordpress\wp-content\plugins\woocommerce-abandoned-cart\includes\libraries\action-scheduler\classes\ActionScheduler_QueueRunner.php:150
[25-Jan-2020 06:00:06 UTC] PHP  13. ActionScheduler_Action->execute() C:\wamp64\www\wordpress\wp-content\plugins\woocommerce-abandoned-cart\includes\libraries\action-scheduler\classes\abstracts\ActionScheduler_Abstract_QueueRunner.php:61
[25-Jan-2020 06:00:06 UTC] PHP  14. do_action_ref_array() C:\wamp64\www\wordpress\wp-content\plugins\woocommerce-abandoned-cart\includes\libraries\action-scheduler\classes\actions\ActionScheduler_Action.php:22
[25-Jan-2020 06:00:06 UTC] PHP  15. WP_Hook->do_action() C:\wamp64\www\wordpress\wp-includes\plugin.php:544
[25-Jan-2020 06:00:06 UTC] PHP  16. WP_Hook->apply_filters() C:\wamp64\www\wordpress\wp-includes\class-wp-hook.php:312
[25-Jan-2020 06:00:06 UTC] PHP  17. Wcal_Process_Base->wcal_process_handler() C:\wamp64\www\wordpress\wp-includes\class-wp-hook.php:288
[25-Jan-2020 06:00:06 UTC] PHP  18. woocommerce_abandon_cart_cron->wcal_send_email_notification() C:\wamp64\www\wordpress\wp-content\plugins\woocommerce-abandoned-cart\includes\wcal_process_base.php:20
